src/cmd/services/m3coordinator/downsample/flush_handler.go

```diff
@@ -67,6 +68,7 @@ func newDownsamplerFlushHandlerMetrics(
 }
 
 func newDownsamplerFlushHandler(
+	tagOptions models.TagOptions,
```

Comment:
nit: Want to make this the second last argument?  We usually tend to put the "options" like args towards the end of the method calls.